### ISSUE TITLE
Make repository host-agnostic

### ISSUE THREAD

**Issue by tdeeb** [teamradish/TRTwitchPlaysBot#28]
A lot of content is stored on the GitHub-specific wiki, such as the getting started section and the syntax tutorial. Move these out of the wiki into separate files so we don't lose them if switching hosts or mirroring the repo elsewhere.

**Comment by tdeeb** [comment]
Completed in https://github.com/teamradish/TRTwitchPlaysBot/commit/256d80974ff3666b6c92ca151dc00426a918e9d5. Some of the direction links are broken, but that'll be fixed once the commit is merged into master.
